exec.go

```diff
@@ -51,7 +51,6 @@ func NewSingleHostQueryExecutor(cfg *ClusterConfig) (e SingleHostQueryExecutor,
 
 	// If protocol version not set assume 4 and skip discovery
 	if c.ProtoVersion == 0 {
-		e.session.logger.Print("gocql: setting protocol version 4")
```

Comment:
Why ?

Reply:
e.session is not created yet, it is nil so this cause panic it is initialized here: https://github.com/scylladb/gocql/blob/master/exec.go#L67

Reply:
Ohh, crazy, so it was broken all this time.